### PR TITLE
example with autofocus

### LINKED ISSUE #100: Add `autofocus` example
cc @valdrinkoshi 

The docs need a little written explanation of how to use this attribute

### PR DESCRIPTION
**Pull request by valdrinkoshi**
Fixes #100.

### CODE REVIEW

**Comment by valdrinkoshi** [comment]
@robdodson @notwaldorf PTAL

File: paper-dialog.html [hunk 1]
```diff
@@ -18,10 +18,11 @@
 `<paper-dialog>` is a dialog with Material Design styling and optional animations when it is
 opened or closed. It provides styles for a header, content area, and an action area for buttons.
 You can use the `<paper-dialog-scrollable>` element (in its own repository) if you need a scrolling
-content area. See `Polymer.PaperDialogBehavior` for specifics.
+content area. Auto focusing can be set to any child using the `autofocus` attribute.
```
Comment (notwaldorf) [inline paper-dialog.html]: How about "To autofocus a specific child element after opening the dialog, give it the `autofocus` attribute"?

**Comment by notwaldorf** [comment]
LGTM, but maybe the sentence can be better? I left a suggestion

**Comment by valdrinkoshi** [comment]
@notwaldorf thanks for the suggestion, feedback incorporated! PTAL

**Pull request merged**